content/articles/how-to-design-logistic-map-using-matlab/index.md

```diff
@@ -0,0 +1,99 @@
+### How to design logistic map using Matlab
+### Introduction
+Let's say you want to model a population of rabbits if you have $X_k$ rabbits this year. How many rabbits will you have next year? The simplest model I can imagine is taking the population $X_k$ and multiplying it by the growth rate $\beta$. Let's say, for example, the growth rate is 2. It means that the population will double every year and the problem with that is the number of rabbits will grow exponentially forever. So here now, we add the term (1-x) to represent the constraints of the environment.
+
```

Comment:
An ntroduction is meant to give readers what to expect. I think you should rework on this intro and don't go into much details as you have done here. Thanks.